mysite/settings.py

```diff
@@ -0,0 +1,119 @@
+"""
+Django settings for mysite project.
+
+Generated by 'django-admin startproject' using Django 2.0.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 's_#$5r*&&pn-p+2@ofesbsgf3pr4ftfy7p#8%c2y=-du@u$6+z'
+
+# SECURITY WARNING: don't run with debug turned on in production!
```

Comment:
Did you read this warning?

Reply:
Yes.

Reply:
So always keep production defaults in repo, while having dev options available.